diff --git a/modules/BrowserGlue.sys.mjs b/modules/BrowserGlue.sys.mjs
index 56aa2ac146..09fa6dcd47 100644
--- a/modules/BrowserGlue.sys.mjs
+++ b/modules//BrowserGlue.sys.mjs
@@ -2788,6 +2788,7 @@ BrowserGlue.prototype = {
 						false,
 					),
 				task: async () => {
+					return;
 					const shellService = Cc[
 						"@mozilla.org/browser/shell-service;1"
 					].getService(Ci.nsIWindowsShellService);
